Cargo.lock

```diff
@@ -36,15 +36,6 @@ dependencies = [
  "xml-rs",
 ]
 
-[[package]]
```

Comment:
whoa, 190 LOC cleared up from our Cargo lock?
I'm eager to recount our dependencies after this :)

Reply:
88 build jobs -> 84 building just wgpu-core